### ISSUE TITLE
Switching between View and ViewModel doesn't work if VM is in different project.

### ISSUE THREAD

**Issue by BenBtg** [mfractor/mfractor-feedback#255]
To have clean seperation of concerns I like to have my ViewModels in a App.Core project which does not have a dependancy on Xamarin.Forms.

I love the ability to quickly switch between View, ViewModel and Codebehind with the Alt+1,2,3 shortcuts.
However it only works if all three files are in the same Project.

##Installation Information

```
--Platform Information--
Operating System: Windows
OS: Win32NT
OS Service Pack: 
OS Version: Microsoft Windows NT 10.0.19041.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.2.2.0
Product Variant: Visual Studio For Windows
Product Variant Version: 16.0.0.0
Product SKU: MFRACTOR-VS-WINDOWS
Commit SHA: 456a155ffb822243ae788e1ff912fa28be4af2e3
Revision: 4363
Branch: master
Date: 12/05/2020 2:26:45 AM (UTC)
Agent: Local
Installed extensions:
	
```

**Comment by ravero** [comment]
Hello @BenBtg, thanks for your feedback.
This is actually a duplicate of the issue #254 
I'll be closing this issue and we'll be dealing your request there.

**Comment by matthewrdev** [comment]
@BenBtg Thanks for reporting this issue. As @ravero has said, this will be resolved by #254, which I am aiming to complete this week.
